### DISCUSSION TITLE
Lolin D1 mini Pro, micro python, Lolin 2.13 ePD

### DISCUSSION THREAD

**Discussion by jonhp** [micropython/micropython#15745] [ESP8266]
Can someone point me to the code and documentation for micro python support of the Lolin 2.13 ePD? I have looked around but not yet found same. Thanks, Jon

**Comment by robert-hh** [comment]
Is that your post: https://forum.micropython.org/viewtopic.php?f=16&t=6609&p=37593#p37593

**Comment by jonhp** [comment]
no, not mine. Jon
